python/tests/test_extractor.py

```diff
@@ -113,8 +113,7 @@ def test_extract_with_snippets(tmpdir: py._path.local.LocalPath) -> None:
         writer.writerow(data)
 
     output = list(extractor.extract_data(str(input_file)))
-    assert 'ceilings' in output[0]
-    assert output[0]['ceilings'][0]['label'] == 'Attic'
+    assert output[0]['ceilings']
```

Comment:
Is this assertion to just check for the existence of a non falsy `ceilings` intentional, or is there a missing `== some_value`?

Reply:
Checking for falsy values (`None`)